### ISSUE TITLE
I have a doubt. When element-plus and tailwindcss exist in nuxt3 at the same time, the color of the element-plus component will appear in a post-rendering process. I don’t know which step is the problem.

### ISSUE THREAD

**Issue by Lucas-lululu** [nuxt/nuxt#22718]
https://github.com/nuxt/nuxt/assets/47276871/8a502cef-7dbd-4dc6-bbd2-5327fde643a4

<img width="372" alt="image" src="https://github.com/nuxt/nuxt/assets/47276871/619327f3-f7d5-4559-880c-5834974f75d0">
<img width="398" alt="image" src="https://github.com/nuxt/nuxt/assets/47276871/d7fc548a-d1ac-4077-9b81-0c0bd86f7e9b">

Here is a code base, hope can help me to see the problem https://github.com/Lucas-lululu/test-code.git

**Comment by ocoke** [comment]
Remove `type="primary"` from the `<el-button>` may solve this problem. I guess the color is the primary color in element-plus?

**Comment by Lucas-lululu** [comment]
> Report content

yes,when I remove the primary, this problem can be removed, but I don’t think this is the fundamental problem, as long as I use tailwindcss, it will appear
<img width="331" alt="image" src="https://github.com/nuxt/nuxt/assets/47276871/227b5494-3073-42c0-9a78-d94a7f752f6b">

**Comment by Lucas-lululu** [comment]
when I comment out this plug-in, the button will not flash. I suspect that it is related to the rendering after reaching the client.
